content/30_deploying_eks_cluster/20_creating_an_eks_cluster.md

```diff
@@ -92,9 +98,10 @@ Outputs:
     kubeconfig: "{\"apiVersion\": \"v1\", \"clusters\": ...}" 
 
 Resources:
-    + 28 created
+    + 27 created
+    27 changes. 31 unchanged
 
 Duration: 14m40s
 
-Permalink: https://app.pulumi.com/workshops/eks-workshop/dev/updates/1
+Permalink: https://app.pulumi.com/workshops/eks-infrastructure/dev/updates/2
```

Comment:
```suggestion
Permalink: https://app.pulumi.com/workshops/eks-infrastructure/dev/updates/1
```

only 1 update has been run thus far

Reply:
We created the VPC so it should be #2 ;)